icefoganalytics/travel-authorization
# Ytgov: Issue 82: Data Modeling for Admin Tables for Travel Rates

Relates to:
- https://github.com/ytgov/travel-authorization/issues/82

# Context

**Is your feature request related to a problem? Please describe.** Every year the travel rates for Meals and Incidentals, distance allowance, and private accommodations are updated. Create admin tables that allow system admin to update these. Allow the ability to audit who updated the field and when the update occured.

**Describe the solution you'd like** Create admin tables in the admin section of the application (access thru top right) for each of the stored amounts. https://yukon.ca/travel-rates

Update the code to hit the newly entered values.

# Implementation

Implement back-end for per-diems and travel allowances.
Standardize per-diems table.
Update bulk generate service to load travel allowances from the database.

# Screenshots

![image](https://github.com/user-attachments/assets/5207e5fd-889c-4169-bd4a-66e0c71324c9)

# Testing Instructions

1. Run the test suite via `dev test` (or `dev test_api`)
2. Boot the app via `dev up`
3. Check that the migrations didn't crash via `dev logs api` in another terminal tab.
4. Check that the seeds run via `dev seed run`
5. Log in to the app at http://localhost:8080
